loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -92,11 +93,17 @@ static void binary_update_download_binary(binary_update_info_t *binary_info, int
 
 	new_version = header_data.bin_ver + 1;
 
-	write_fd = binary_manager_open_new_entry(APP_NAME, new_version);
+	ret = binary_manager_get_download_path(APP_NAME, new_version, new_filepath);
+	if (ret < 0) {
+		fail_cnt++;
+		printf("binary_manager_get_download_path FAIL %d,\n", ret);
+		return;
```

Comment:
hmm...I think you had better return type of this function, because after call this function, binary_update_invalid_binary_test  try reload... You think is it safe? how do you think?

Reply:
done